grpc/stretch/arm32v7/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM arm32v7/debian:stretch-slim as build-env
 
 COPY ./qemu-arm-static /usr/bin/
 
-ENV GRPC_VERSION=v1.20.x
+ENV GRPC_VERSION=v1.27.x
```

Comment:
Should this be v2.27?

Reply:
It seems that only the dotnet grpc extension was updated to 2.x in GRPC 1.23 (https://github.com/grpc/grpc/releases/tag/v1.23.0).

The actual GRPC project tags are still 1.x https://github.com/grpc/grpc/releases/tag/v1.27.3